test/unit/presenters/publishing_api/fatality_notice_presenter_test.rb

```diff
@@ -119,6 +119,10 @@ class PublishingApi::FatalityNoticePresenterDetailsTest < ActiveSupport::TestCas
       body: "*Test string*",
     )
 
+    2.times do |i|
+      create(:fatality_notice_casualty, fatality_notice: @fatality_notice, personal_details: "personal details #{i}")
```

Comment:
Just FYI, you can do this in the factory itself by using `sequence` which means a loop wouldn't be needed here (you could just have create listed twice), i.e. replace `personal_details { "personal-details" }` with `sequence(:personal_details) { |i| "personal details #{index}" }`

Reply:
good shout, didn't think when i copied this from the other pr which was a bit more of a complicated set up, sorted

Reply:
sorted!